vnext/Microsoft.ReactNative/ABIViewManager.cpp

```diff
@@ -215,6 +216,37 @@ YGMeasureFunc ABIViewManager::GetYogaCustomMeasureFunc() const {
   }
 }
 
+bool ABIViewManager::OnPointerPressed(const xaml::Input::PointerRoutedEventArgs &args) {
+  return m_viewManagerWithPointerEvents ? m_viewManagerWithPointerEvents.OnPointerPressed(args)
+                                        : Super::OnPointerPressed(args);
+}
+
+bool ABIViewManager::OnPointerMoved(const xaml::Input::PointerRoutedEventArgs &args) {
+  return m_viewManagerWithPointerEvents ? m_viewManagerWithPointerEvents.OnPointerMoved(args)
+                                        : Super::OnPointerMoved(args);
+}
+
+bool ABIViewManager::OnPointerReleased(const xaml::Input::PointerRoutedEventArgs &args) {
+  return m_viewManagerWithPointerEvents ? m_viewManagerWithPointerEvents.OnPointerReleased(args)
+                                        : Super::OnPointerReleased(args);
+}
+
+void ABIViewManager::OnPointerCanceled(const xaml::Input::PointerRoutedEventArgs &args) {
+  if (m_viewManagerWithPointerEvents) {
```

Comment:
The interface needs to also accept the ShadowNode, and the IDL interface likely needs to take a xaml::DependencyObject / XamlView.